zerver/lib/upload.py

```diff
@@ -205,8 +205,16 @@ def upload_avatar_image_local(user_file, user_profile, email):
 
 def claim_attachment(path_id, message):
     #update attachment entry in DB
-    Attachment.objects.filter(path_id=path_id).update(message=message, is_claimed=True)
-    return True
+    try:
+        Attachment.objects.filter(path_id=path_id).update(message=message)
+        return True
+
+    except Attachment.DoesNotExist:
+        #This error msg should be propogated to the browser
+        print("The upload was not successful. Please reupload the file again")
```

Comment:
@timabbott  Ideally this error message should be displayed in the browser. You use json_error to send error messages but is it possible to use the same inside the do_send_messages function in action.py? The claim_attachment function is called by it.